universal-application-tool-0.0.1/app/services/applicant/Block.java

```diff
@@ -62,4 +62,9 @@ public boolean hasErrors() {
     }
     return errorsMemo.get();
   }
+
+  @Override
+  public String toString() {
+    return "Id: " + this.id;
```

Comment:
suggestion (blocking): include the class name here e.g. `'Block [id=" + this.id + "]"`

Reply:
Much better, thank you!